src/main/java/net/testudobank/TestudoBankRepository.java

```diff
@@ -102,6 +102,11 @@ public static void deleteRowFromOverdraftLogsTable(JdbcTemplate jdbcTemplate, St
     jdbcTemplate.update(deleteRowFromOverdraftLogsSql);
   }
 
+  public static int applyInterestRateToPennyAmount(int pennyAmount, final double INTEREST_RATE) {
```

Comment:
You don't need to take `INTEREST_RATE` as a parameter for this method. You can just use `public final static double INTEREST_RATE = 1.02;` variable defined at the top of this file.

Reply:
Also, it would be better to put this helper method in MvcController.java itself. The `Repository.java` file is reserved mainly for helper methods that wrap SQL queries to the database